include/sound/sof/dai-intel.h

```diff
@@ -88,6 +88,8 @@ struct sof_ipc_dai_ssp_params {
 struct sof_ipc_dai_hda_params {
 	struct sof_ipc_hdr hdr;
 	uint32_t link_dma_ch;
+	uint32_t rate;
```

Comment:
@bardliao Please add a short summary of why the change is done why it doesn't break the ABI. This doesn't have to be detailed explanation, but I think a bit more is warranted in the kernel side commit. Maybe something like: "FW interface for HDA DAI parameters was extended with information on sampling rate and channel count in version 3.16. Align kernel header with the FW change. This change is backwards compatible. Old firmware will ignore the values."